app/tests/api/test_auth_signup_otp.py

```
@@ -1,80 +1,109 @@
+    assert r1.status_code == 204, r1.text
+
+    # wrong otp
+    r2 = client.post("/api/v1/auth/verify-otp", json={"email": email, "otp": "000000"})
+    assert r2.status_code == 422, r2.text
+
+
+def test_request_otp_email_rate_limit(monkeypatch, client):
+    limiter = next(get_rate_limiter())
+    call_counts = {"email": 0}
+
+    async def fake_allow_request(*, key: str, limit: int, window_seconds: int):
```

Ruff (0.14.14)

[warning] Unused function argument: `limit`

(ARG001)